index.js

```diff
@@ -0,0 +1,37 @@
+function calculateSwitch(numbers) {
+  switch (numbers[1]) {
+    case '+':
+      return numbers[0] + numbers[2]
+    case '-':
+      return numbers[0] - numbers[2]
+    case '*':
+      return numbers[0] * numbers[2]
+    case '/':
+      return numbers[0] / numbers[2]
```

Comment:
You should be typecasting these before doing the calculation. Right now the value `'2'` would make it here and `'2' + 2` is `'22'` not `4` so you'd get weird behavior.